cmd/manager/main.go

```diff
@@ -59,6 +62,17 @@ func main() {
 		log.Fatal(err)
 	}
 
+	// Set proxy API availability
+	k8sInterface, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = proxy.SetProxyAvailability(k8sInterface.Discovery())
+	if err != nil {
+		log.Fatal(err)
```

Comment:
This method will only attempt to set proxy availability if the Proxy API exists on the cluster. Since CVO is managing the cluster proxy and marketplace, the operator will eventually resolve.

Reply:
if you can't get a config it makes sense to me to error hard, we wouldn't be able to contact k8s and I bet that `manager.New` would also fail and you would log.Fatal on line 82. 

The question for setting the proxy availability, I think is more nuanced, what are the error scenarios for this and does it make sense, that if something were to go wrong you would want the operator to continue.